### PR TITLE
State clearing browser cache and session as required upon upgrading

### PR DESCRIPTION
**Pull request by madhuracj**
Signed-off-by: Madhura Jayaratne madhura.cj@gmail.com

### CODE REVIEW

File: doc/setup.rst [hunk 1]
```diff
@@ -436,6 +436,9 @@ newer (<= 4.2.x) and if you use the phpMyAdmin configuration storage, you
 should run the :term:`SQL` script found in
 :file:`sql/upgrade_column_info_4_3_0+.sql`.
 
+Do not forget to clear the browser cache and to empty the old session by
+loging out and login again.
```
Comment (ibennetch) [inline doc/setup.rst]: "logging out and logging in again." is probably better here.

**Comment by madhuracj** [comment]
Thanks @ibennetch. I've updated the wording in the new commit.

**Comment by ibennetch** [comment]
Thanks :)

**Comment by lem9** [comment]
Thanks; too bad we cannot automate these actions.

**Pull request merged**